bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -69,6 +71,41 @@ This has the following advantages:
 If you want electricity tax included in your total price, please add either `electricity-tax` or `reduced-electricity-tax` to the group - depending on which one applies.
 See [Electricity Tax](#electricity-tax) for further information.
 
+##### Time Series
+
+Group items with aggregate functions are not automatically recalculated into the future when time series for child items are updated.
+Therefore, the `SUM` function mentioned above will only work for the current price.
+Calculation of future total prices can be achieved with a rule:
+
+:::: tabs
+
+::: tab JavaScript
+
+```javascript
+rules.when()
+    .channel('energidataservice:service:energidataservice:electricity#event').triggered('DAY_AHEAD_AVAILABLE')
+    .then(event => {
+        var timeSeries = new items.TimeSeries('REPLACE');
+        var start = time.LocalDate.now().atStartOfDay().atZone(time.ZoneId.systemDefault());
+        var spotPrices = items.SpotPrice.persistence.getAllStatesBetween(start, start.plusDays(2));
+        for (var spotPrice of spotPrices) {
+            var totalPrice = spotPrice.quantityState
+                .add(items.GridTariff.persistence.persistedState(spotPrice.timestamp).quantityState)
```

Comment:
Looks good now!

Just one minor remark:

```suggestion
                .add(items.GridTariff.persistence.persistedState(spotPrice.timestamp))
```

Should work as well.

Reply:
Hmm, how? `Item.persistence.persistedState(timestamp)` returns `PersistedItem` according to documentation. `PersistedItem` contains the properties `state`, `numericState`, `quantityState`, `rawState` and `timestamp`. I need a `Quantity` to add it to `spotPrice.quantityState` which is also `Quantity`. Still, maybe I'm missing something, so I tried with this result:

```
org.graalvm.polyglot.PolyglotException: TypeError: Argument of wrong type provided, required Item, string or Quantity.
```

Reply:
Oh sorry, I thought that you got a normal Item there (probably too late to play around with code now).